frontend/src/metabase/querying/components/FilterPanel/FilterPanelButton/FilterPanelButton.styled.tsx

```diff
@@ -0,0 +1,26 @@
+import type { ButtonHTMLAttributes } from "react";
+import styled from "@emotion/styled";
+import { alpha, color } from "metabase/lib/colors";
+import { Button } from "metabase/ui";
+import type { ButtonProps } from "metabase/ui";
+
+type FilterButtonProps = ButtonHTMLAttributes<HTMLButtonElement> &
+  ButtonProps & {
+    isExpanded: boolean;
+  };
+
+export const FilterButton = styled(Button)<FilterButtonProps>`
+  color: ${({ isExpanded }) => (isExpanded ? color("white") : color("filter"))};
+  background-color: ${({ isExpanded }) =>
+    isExpanded ? alpha("filter", 0.8) : alpha("filter", 0.2)};
+  transition: border 300ms linear, background 300ms linear;
```

Comment:
I see this code has just been moved but what about color?

I'd just go with `all` and optimize by listing all necessary properties only if there are performance issues (but I don't think a button will suffer from those).

Reply:
It seems that we don't want `color` to be transitioned. At least it looks noticeably different if we do so. That's why I copied it as it was.